src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ConfigDebugCommandTest.php

```diff
@@ -58,6 +59,16 @@ public function testParametersValuesAreResolved()
         $this->assertStringContainsString('secret: test', $tester->getDisplay());
     }
 
+    public function testDefaultParameterValueIsResolvedIfConfigIsExisting()
+    {
+        $tester = $this->createCommandTester();
+        $ret = $tester->execute(['name' => 'framework']);
+
+        $this->assertSame(0, $ret, 'Returns 0 in case of success');
+        $kernelCacheDir = self::$kernel->getContainer()->getParameter('kernel.cache_dir');
```

Comment:
`$this->application->getKernel()` would work isnt it? Let's try to not introduce extra state :)